backend/open_webui/main.py

```diff
@@ -2356,7 +2353,7 @@ async def oauth_callback(provider: str, request: Request, response: Response):
     )
 
     # Redirect back to the frontend with the JWT token
-    redirect_url = f"{request.base_url}auth#token={jwt_token}"
+    redirect_url = f"http://localhost:5173/auth#token={jwt_token}&id_token={id_token}"
```

Comment:
Why are we redirecting to localhost:5173 and not request.base_url ?

Reply:
reverted